Makefile

```diff
@@ -120,51 +142,68 @@ DEPENDS	:=	$(OFILES:.o=.d)
 #---------------------------------------------------------------------------------
 # main targets
 #---------------------------------------------------------------------------------
-all: $(OUTPUT).gcb $(OUTPUT).vgc $(OUTPUT).gci $(OUTPUT)_xz.gci
 
+#---------------------------------------------------------------------------------
+# Base DOL (PicoBoot)
```

Comment:
Nit: not really specific to PicoBoot since every other output format is derived from this. I shipped plain DOLs before PicoBoot even existed (though I'm not sure what their purpose would have been :stuck_out_tongue:).

Reply:
Oops. That is actually a leftover artifact from when I had the comments on the alias targets. I was trying to convey "Used for devices such as PicoBoot" with the parenthesis. Will remove.

Reply:
In this case, you could just put a comment next to the aliases to document that. Better yet, make picoboot an alias of dol.

Reply:
I am working on updates to the README and plan to have a section for each of the platforms including PicoBoot. That should suffice. As you say there is nothing specific to PicoBoot and I wouldn't want to give the impression that `make picoboot` is doing something special.